client/plan/update-routes.js

```diff
@@ -153,7 +153,8 @@ function updateRoutes(plan, opts, callback) {
         event_type: 'query',
         url: location.href,
         //results: JSON.stringify(data),
-        client_ip: "000.111.000.111",//app.get('client_ip'),
```

Comment:
This has never worked properly and was improperly merged to master sometime ago

Any chance we can easily fix this with a MAC ID ? @bijanmmarkes

Reply:
Not sure if it is an easy fix; As it most certainly will have to be done on the server-side of things, and populated back to the client; And as you know, it would be great if we could do this with a NodeJS back-end module which actually would make it super easy; but unfortunately Component JS is restricting us from those capabilities. @linuxwebexpert